sycl/test/vitis/simple_tests/memory_types_test.cpp

```diff
@@ -0,0 +1,43 @@
+// REQUIRES: vitis
+
+// RUN: rm -rf %t.dir && mkdir %t.dir && cd %t.dir
+// RUN: %clangxx %EXTRA_COMPILE_FLAGS-fsycl -std=c++20 -fsycl-targets=%sycl_triple %s -o %t.dir/exec.out > %t.check 2>&1
+// RUN: %run_if_not_cpu FileCheck --input-file=%t.check %s
+// RUN: %ACC_RUN_PLACEHOLDER %t.dir/exec.out
+
+#include <sycl.hpp>
```

Comment:
```suggestion
#include <sycl/sycl.hpp>
```
@Ralender curious. I think upstream was changed to avoid such a thing to compile. Perhaps a bad merge conflict? Or perhaps it is already fixed in https://github.com/triSYCL/sycl/pull/216 ?

Reply:
There is still 300+ uses of 
`#include <sycl/sycl.hpp>`
so I dont think there as a serious attempt are avoid it.

Reply:
Curious, I think I have seen a removal of `#include <sycl.hpp>` upstream with a CMake change to avoid such examples to compile.

Reply:
oh I mis-undrsetood what you meant.
`#include <sycl.hpp>`
is indeed barely used anymore.